docs/source/howto/data.rst

```diff
@@ -101,15 +101,15 @@ Once stored, this data can always be retrieved through the ``source`` property:
 
 The following list shows all the keys that are allowed to be set in the ``source`` dictionary:
 
-    * 'db_name': The name of the external database.
-    * 'db_uri': The base URI of the external database.
-    * 'uri': The exact URI of where the data can be retrieved. Ideally this is a persistent URI.
-    * 'id': The external ID with which the data is identified in the external database.
-    * 'version': The version of the data, if any.
-    * 'extras': Optional dictionary with other fields for source description.
-    * 'source_md5': MD5 checksum of the data.
-    * 'description': Human-readable free form description of the data's source.
-    * 'license': A string with the type of license that applies to the data, if any.
+* 'db_name': The name of the external database.
+* 'db_uri': The base URI of the external database.
+* 'uri': The exact URI of where the data can be retrieved. Ideally this is a persistent URI.
+* 'id': The external ID with which the data is identified in the external database.
+* 'version': The version of the data, if any.
+* 'extras': Optional dictionary with other fields for source description.
+* 'source_md5': MD5 checksum of the data.
+* 'description': Human-readable free form description of the data's source.
+* 'license': A string with the type of license that applies to the data, if any.
```

Comment:
Since you are already here. I don't think single quotes are needed here as the name of the keys.

Reply:
You want me to change them to backticks to make them inline code, or just remove the quotes entirely? Think backticks would be good, right?

Reply:
I have no preference, to be honest, I think just removing the quotes is clear enough.

Reply:
Had another look and since they are keys in the database, I think double backticks are more appropriate.